pkg/platform/kube/controller/nucliofunction.go

```diff
@@ -257,13 +257,25 @@ func (fo *functionOperator) CreateOrUpdate(ctx context.Context, object runtime.O
 		if err, functionState := fo.functionresClient.WaitAvailable(waitContext,
 			function,
 			functionResourcesCreateOrUpdateTimestamp); err != nil {
+
+			// Update the function selector if necessary even if the function previously scaled from zero and waiting for availability failed,
```

Comment:
what do you mean by `even`? its only updated if it scaling from zero...

Reply:
I think I had `only` in my head here, will fix